[s390x]periodic-kubevirt-e2e-test-S390X job is failing

<!-- Make sure that you visit our User Guide at https://kubevirt.io/user-guide.
-->

**What happened**:
A clear and concise description of what the bug is.
The s390x periodic-kubevirt-e2e-test-S390X  is failing with the following error
```
04:34:08: INFO: Analyzed target //:gazelle (0 packages loaded, 0 targets configured).
04:34:08: INFO: Found 1 target...
04:34:09: Target //:gazelle up-to-date:
04:34:09:   bazel-bin/gazelle-runner.bash
04:34:09:   bazel-bin/gazelle
04:34:09: INFO: Elapsed time: 0.857s, Critical Path: 0.57s
04:34:09: INFO: 1 process: 1 internal.
04:34:09: INFO: Running command line: bazel-bin/gazelle --exclude kubevirtci/cluster-up
04:34:09: qemu-s390x-static: Could not open '/lib/ld64.so.1': No such file or directory
make: *** [Makefile:37: bazel-build-images] Error 1
```
Ref:
https://storage.googleapis.com/kubevirt-prow/logs/periodic-kubevirt-e2e-test-S390X/1986289948084408320/build-log.txt


**What you expected to happen**:
It is expected to run successfully without any error.

**How to reproduce it (as minimally and precisely as possible)**:
Steps to reproduce the behavior.

**Additional context**:
Add any other context about the problem here.

**Environment**:
- KubeVirt version (use `virtctl version`): N/A
- Kubernetes version (use `kubectl version`): N/A
- VM or VMI specifications: N/A
- Cloud provider or hardware configuration: N/A
- OS (e.g. from /etc/os-release): N/A
- Kernel (e.g. `uname -a`): N/A
- Install tools: N/A
- Others: N/A

--- a/hack/bootstrap.sh
+++ b/hack/bootstrap.sh
@@ -41,7 +41,7 @@ function kubevirt::bootstrap::regenerate() {
         rm .bazeldnf/sandbox.bazelrc -f
         # Run gazelle to ensure that nogo has all build files resolved and that we can bootstrap the env.
         # This is necessary since some steps remove the vendor build files and nogo would be broken then.
-        KUBEVIRT_BOOTSTRAPPING=true bazel run --config=${ARCHITECTURE} //:gazelle -- --exclude kubevirtci/cluster-up
+        KUBEVIRT_BOOTSTRAPPING=true bazel run --config=${HOST_ARCHITECTURE} //:gazelle -- --exclude kubevirtci/cluster-up
         KUBEVIRT_BOOTSTRAPPING=true bazel run --config ${HOST_ARCHITECTURE} //rpm:sandbox_${1}
         bazel clean
 

